packages/story-editor/src/components/library/panes/media/common/insertionMenu.js

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import {
+  useRef,
+  useMemo,
+  useCallback,
+  useState,
+} from '@googleforcreators/react';
+import { __ } from '@googleforcreators/i18n';
+import { v4 as uuidv4 } from 'uuid';
+import {
+  Button,
+  BUTTON_VARIANTS,
+  Icons,
+  Menu,
+  PLACEMENT,
+  Popup,
+} from '@googleforcreators/design-system';
+import { getSmallestUrlForWidth } from '@googleforcreators/media';
+
+/**
+ * Internal dependencies
+ */
+import getElementProperties from '../../../../canvas/utils/getElementProperties';
+import useStory from '../../../../../app/story/useStory';
+import useNestedRovingTabIndex from '../../shared/hooks/useNestedRovingTabIndex';
+
+const AddButton = styled(Button).attrs({ variant: BUTTON_VARIANTS.ICON })`
+  display: flex;
+  align-items: center;
+  position: absolute;
+  top: calc(50% - 16px);
+  right: calc(50% - 16px);
+  color: ${({ theme }) => theme.colors.fg.primary};
+  border-radius: 100%;
+  width: 28px;
+  height: 28px;
+  opacity: ${({ display }) => (display ? '1' : '0')};
+`;
+
+const IconContainer = styled.div`
+  height: 32px;
```

Comment:
any value in this being a const and then using it divided by 2 to get your 16px on line 51/52 so that the sizing and positioning stay connected easily?